librz/core/canalysis.c

```diff
@@ -5560,29 +5560,39 @@ static void sdb_concat_by_path(Sdb *s, const char *path) {
 	sdb_free(db);
 }
 
-RZ_API void rz_core_analysis_cc_init(RzCore *core) {
+RZ_API bool rz_analysis_cc_init(RzCore *core, RZ_NULLABLE const char *path) {
```

Comment:
if you want this to be only rz_analysis, the it should be moved under the analysis directory, not under core.